va-gov/pages/careers-employment/index.md

```diff
@@ -182,7 +182,7 @@ We can support you in all stages of your job search—from returning to work wit
   </div>
 
   <div class="link">
-    <a href="https://linkedinforgood.linkedin.com/programs/veterans"><b>Get Free Classes for a Year (LinkedIn)</b></a>
+    <a href="https://linkedinforgood.linkedin.com/programs/veterans" target="_blank"><b>Get Free Classes for a Year (LinkedIn)</b></a>
```

Comment:
We should add `rel="noopener"` to this link, since it's going out to a commercial site that we don't control.

Reply:
@jbalboni Didn't know we needed that. Thanks. Updated now.